probcal/data_modules/coco_people_datamodule.py

```diff
@@ -124,15 +125,23 @@ def __init__(
     ):
         super().__init__(root_dir, batch_size, num_workers, persistent_workers, surface_image_path)
 
-    def setup(self, stage):
+    def _get_ood_transform(self, **kwargs):
+        # implemented by base class
```

Comment:
Should raise a `NotImplementedError` if this gets called on a subclass that doesn't override this method

Reply:
Good catch. I ended up removing this entirely and just created different subclasses, each with a different `setup()`.  Due to how the MixUpTransform works, this ended up being easier. 

Removed.